packages/lpa-questionnaire-web-app/src/views/includes/head.njk

```diff
@@ -0,0 +1,17 @@
+{% block head %}

```

Comment:
there's a lot of debate around this, so heads up, this may not remain as is

Reply:
can you keep us updated or if we can be party to discussions that would be helpful. If you can give more details as to what is up for debate that would help also.

Reply:
Sure - I'm not actively working on GA stuff so this was to flag that there's a potential for this to change but I may not have immediate visibility on it.

The gist is that this code drops a GA cookie on every user, whereas we should probably have a cookie banner which needs accepting before we drop the cookie. That's all still up for debate afaik, but if it impacts us, it will likely impact you, too. 

No action at this time ofc, just a heads up.

Reply:
That's great, thanks for the extra detail. Will leave this as-is for now